src/main/java/uk/gov/companieshouse/api/strikeoffobjections/model/patcher/ObjectionPatcher.java

```diff
@@ -0,0 +1,22 @@
+package uk.gov.companieshouse.api.strikeoffobjections.model.patcher;
+
+import org.springframework.stereotype.Component;
+import uk.gov.companieshouse.api.strikeoffobjections.model.entity.Objection;
+import uk.gov.companieshouse.api.strikeoffobjections.model.request.ObjectionRequest;
+
+@Component
+public class ObjectionPatcher {
+
+    public Objection patchObjection(ObjectionRequest objectionRequest, String requestId, Objection existingObjection) {
```

Comment:
Rather than 'ObjectionRequest', could the object this class/method accepts be called 'ObjectionPatch' instead? We were trying to reserve the word 'request' only for HTTP request related things.